docgen/src/guides/synchronize-multiple-instantsearch.md

```diff
@@ -0,0 +1,89 @@
+---
+title: Using multiple InstantSearch instances
+layout: guide.pug
+category: guide
+navWeight: 4
+---
+
+Having several instances of `InstantSearch` can be useful when: 
+
+* You want to make a search on multiple indices 
+* You want to synchronize multiple `InstantSearch` instances
+
+To be able to do that, we are going to take advantage of two props on the [InstantSearch root component](/component/InstantSearch.html).
+
+* onStateChange(nextState): a function being called every time the `InstantSearch` state is updated. 
+* state: an object that is the current state of InstantSearch
+
+The idea is to have a main component that will receive every new state of each instances and create a unique one from them.
+Then, this state will be passed back to each `InstantSearch` instances.  
+
+Be aware that just having a refinement inside the `InstantSearch` state doesn't apply it automatically to the search. 
+To apply the refinement, you will need to use a [virtual widget](/guides/advanced-topics.html#how-to-preselect-values-using-virtual-widgets). 
+
+Here's an example showing you how to make the same search on two different indices: 
+ 
+```jsx
+import React, {Component} from 'react';
+import { InstantSearch, Hits, SearchBox } from 'react-instantsearch/dom';
+import {connectSearchBox} from 'react-instantsearch/connectors'
+
+class App extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {searchState: {}};
+  }
+
+  onStateChange = (nextState) => {
+    //we merge both InstantSearch search state in one
+    this.setState({searchState: {...this.state.searchState, ...nextState}});
```

Comment:
I think the main usecase is to show hits from a different index, so there's no need to READ the state of the second instance since there's no clear usecase on why we would like to read the changes of an instance that we use only to display, not allowing refinements.

I would take the opportunity to remove the constraint of passing both `state=` and `onStateChange`.

What do you think?